src/audiomixerboard.cpp

```diff
@@ -1123,7 +1123,7 @@ void CAudioMixerBoard::ChangeFaderOrder ( const EChSortType eChSortType )
             }
             break;
         case ST_BY_SERVER_CHANNEL:
```

Comment:
Should this get renamed if it's always the received MIDI ID?

Reply:
I did wonder about that, and even started to change the name, but then had second thoughts, as it implied the menu option should change to "Sort by MIDI Channel". I think this could be more confusing to some users.

The actual server channel numbers and the channel assigned to the particular client are never actually visible to the user, except when used for MIDI channels.

What we are doing here by mapping is presenting to the user a view that _always_ uses `0` for the user's own channel, and maps the rest of the channels up by one. As far as the user is concerned, they _are_ server channels (even if virtual); we just guarantee that the user themselves is channel zero. So whether or not a MIDI controller is used, a "Sort by Server Channel" will always put the user's own channel first, whether or not "Show Own Fader First" is enabled. I think that is a worthwhile improvement.

Reply:
I think that's a definite source of potential confusion, to be honest.  I'd _only_ want my fader sorted first if I asked for it, if there's an option in the menu.  I'd be wondering why the option was there, otherwise.

Certainly for "No sorting", always putting own fader first is wrong unless requested.